eng/common/mcp/azure-sdk-mcp.ps1

```diff
@@ -6,6 +6,7 @@ param(
     [string]$Version, # Default to latest
     [string]$InstallDirectory = (Join-Path $HOME ".azure-sdk-mcp" "azsdk"),
     [string]$Repository = 'Azure/azure-sdk-tools',
+    [string]$RunDirectory = ($PSScriptRoot + "/../../../"),
```

Comment:
FWIW https://github.com/Azure/azure-sdk-tools/blob/d008f93d062cea221e32f2e5d2ac5b4f7feb371b/eng/common/scripts/common.ps1#L3

That is how we define this in common.

Reply:
Looks like we also resolve a similar path to the mcp.josn file below.

Reply:
> FWIW
> 
> https://github.com/Azure/azure-sdk-tools/blob/d008f93d062cea221e32f2e5d2ac5b4f7feb371b/eng/common/scripts/common.ps1#L3
> 
> That is how we define this in common.

This is actually bugged, you need a leading slash. Hence why it's using 4 `..` instead of 3 (the first ends up getting ignored).

Reply:
https://github.com/Azure/azure-sdk-tools/pull/10694